calico_kubernetes/logutils.py

```diff
@@ -33,11 +33,14 @@ def configure_logger(logger, log_level, log_format=LOG_FORMAT,
         os.makedirs(log_dir)
 
     formatter = logging.Formatter(log_format)
+    docker_filter = IdentityFilter(identity=filter_id)
```

Comment:
This is awkward, since we're calling it docker_filter, but the argument is "identity" which is generic and says nothing about the docker_id.

We should probably just be specific and call it "docker_id" so it is clear what we mean - we're never passing anything else in, right?